script/README.md

```diff
@@ -12,20 +12,23 @@ To use these scripts, please follow these steps **first**:
 - [ ] If you're using GitHub Enterprise, and access to github.com is limited within your organization, import the following repos and all their branches in your newly created organization: [caption-this](https://github.com/githubtraining/caption-this), [polygons](https://github.com/githubtraining/polygons), [github-games](https://github.com/githubtraining/github-games). You'll need access to GitHub.com to perform this one-time operation.
   - _Note: For the [caption-this](https://github.com/githubtraining/caption-this) repository to function properly, the `baseurl` parameter in `_config.yml` needs to be the exact same as the GH pages link generated by GHE._
 - [ ] Ensure your user account is an owner of the newly created training org, and add any other trainers as owners of the org. Alternatively, you can create a separate teaching account that all trainers have access to, and make that teaching account an owner of the org.
-- [ ] Create a [Personal Access Token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) for your user account, or the shared teaching account. You'll need this for the [local configuration](#local-configuration).
+- [ ] Create a [Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) for your user account, or the shared teaching account. You'll need this for the [local configuration](#local-configuration).
+  - _Note: Your personal access token must include the scopes **repo** and **delete_repo**._
+- OPTIONAL: If you want to collect feedback from the class, have a URL ready. You can use services such as SurveyMonkey or Google Forms.
 - OPTIONAL: If you're going to offer participants one-on-one appointments, determine the URL participants will use to schedule those. We use [YouCanBook.me](http://youcanbook.me). You'll need the URL later.
 
 ### Local Configuration
 
 Generally speaking, we'll create a `.trainingmanualrc` in your home directory, and source that file from your Bash or ZSH profile. The scripts take care of doing that for you. When performing an operation using the training scripts, the variables specified in your `~/.trainingmanualrc` file will be used.
 
-Here's how to get your local environment up and running: 
+Here's how to get your local environment up and running:
 
 1. install [jq](https://stedolan.github.io/jq/download/)
 2. run `script/teach-class` and choose option 0 to set up your environment variables. You'll need the following pieces of information from [above](#remote-configuration):
    - the title of your newly created training organization
    - your username, or the username of a shared teaching account
-   - your personal access token (PAT) or the PAT of the shared teaching account
+   - your personal access token (PAT) or the PAT of the shared teaching account that includes the scopes **repo** and **delete_repo**.
```

Comment:
Same as above, reminder regarding authorization

Reply:
Same as above, reopening it

Reply:
@EduSemensati it's been forever, this month went by fast. I finally got around updating it!